tests/models/test_category.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import json
+
+from flask_sqlalchemy import SQLAlchemy
+from app import app
+from models import Category
+
+class CategoryTestCase(unittest.TestCase):
+  def setUp(self):
+    self.client = app.test_client()
+    self.db = SQLAlchemy(app)
+
+    self.category=Category(
+        title="Definition of Domestic Violence",
+        active=True,
+    )
+
+    self.db.session.add(self.category)
+    self.db.session.commit()
+
+  def tearDown(self):
+    self.db.session.query(Category).delete()
+    self.db.session.commit()
+
+  def test_init(self):
+    self.assertEqual(self.category.title, "Definition of Domestic Violence")
+    self.assertTrue(self.category.active)
+
+  def test_serialize(self):
+    self.assertDictContainsSubset(
+      {
+        "title": "Definition of Domestic Violence",
+        "active": True
+      },
+      self.category.serialize()
+    )
```

Comment:
Ugghh why did they deprecate `assertDictContainsSubset`? This test looks so clean as it is right now :(

Reply:
😭 😭 😭  I'm gonna use it until they force me not to lmao